config/august/asl-03.xml

```diff
@@ -33,8 +33,21 @@ time.</MetaDataItem>
     <MetaDataItem id="0001" name="FrequencyName" type="0001">U.S. / Canada / Mexico</MetaDataItem>
     <ChangeLog>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="02 Jun 2019" revision="2">Initial Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/2624/xml</Entry>
+      <Entry author="Quinn Hosler - https://github.com/quinnhosler" date="26 April 2020" revision="3">DoorSense(TM) support</Entry>
     </ChangeLog>
   </MetaData>
+    <!-- Configuration Parameters -->
+  <CommandClass id="98">
+    <Value genre="config" type="list" index="7" instance="1" label="DoorSense" min="0" max="3" size="1" value="2">
+      <Help>
+       State of Door and Bolt
+      </Help>
+      <Item label="Locked/Open" value="0" />
+      <Item label="Locked/Closed" value="1" />
+      <Item label="Unlocked/Open" value="2" />
+      <Item label="Unlocked/Closed" value="3" />
+    </Value>
+  </CommandClass>
```

Comment:
This is not a Configuration CC id - Why is it required?

Reply:
You're right this isn't needed and I didn't understand why I was making that change. I removed the changes to this file.